fleet/comms_rpc.go

```diff
@@ -62,8 +62,9 @@ type GroupRemovedRPC struct {
 }
 
 type GroupRemovedRPCPayload struct {
-	AgentGroupID string `json:"agent_group_id"`
-	ChannelID    string `json:"channel_id"`
+	AgentGroupID string   `json:"agent_group_id"`
+	ChannelID    string   `json:"channel_id"`
+	DatasetsID   []string `json:"datasets_id"`
```

Comment:
please check for precedents when creating new fields in a public interface - it looks like we already use "datasets", not "datasets_id" when communicating an array of dataset IDs

https://github.com/ns1labs/orb/blob/328b48b4f71189e0557d2f7a2753b7b9d0fb8dde/fleet/comms_rpc.go#L132